packages/ove-app-networks/src/client/constants/networks.js

```diff
@@ -4,11 +4,63 @@ const Constants = {
     **************************************************************/
     DEFAULT_STATE_NAME: 'SigmaSample',
 
+    /**************************************************************
+                               Back-end
+    **************************************************************/
+    SOCKET_REFRESH_DELAY: 5000, // Unit: milliseconds
+    SOCKET_READY_WAIT_TIME: 3000, // Unit: milliseconds
+    HTTP_HEADER_CONTENT_TYPE: 'Content-Type',
+    HTTP_CONTENT_TYPE_JSON: 'application/json',
+
     /**************************************************************
                                 Common
     **************************************************************/
     CONTENT_DIV: '#graphArea',
     APP_NAME: 'networks'
 };
 
+/**************************************************************
+                            Enums
+**************************************************************/
+Constants.Operation = {
+    SEARCH: 'search',
+    COLOR: 'color',
+    LABEL: 'labelNodes',
+    NEIGHBORS_OF: 'neighborsOf'
+};
+
+Constants.Evaluation = {
+    PROPERTY: 'property',
+    LITERAL: 'literal',
+    FUNCTION_CALL: 'functionCall'.toLowerCase(),
```

Comment:
can this be simplified?

Reply:
I'm assuming the simplification is to get rid of the `toLowerCase` conversion, which is also questioned below. I have addressed that there, and therefore resolving this comment.